config/migrate.ts

```diff
@@ -130,8 +130,12 @@ export function mergeConfigProperties(
   const conflicts: Array<ConflictProperty> = [];
 
   propertiesToCheck.forEach(prop => {
-    if (prop in globalConfig && prop in deprecatedConfig) {
-      if (force || globalConfig[prop] === deprecatedConfig[prop]) {
+    if (prop in deprecatedConfig) {
+      if (
+        force ||
+        !(prop in globalConfig) ||
+        globalConfig[prop] === deprecatedConfig[prop]
+      ) {
```

Comment:
Could be some tests to cover this function and make sure we don't add regressions?

Reply:
I'll add tests as a fast follow--I want to get this out today before tomorrow's launch of global config.